os/tools/check_package_size.py

```diff
@@ -113,8 +113,9 @@ def check_binary_header(bin_type):
 COMMON_PART_TMP_SIZE = FLASH_SIZE
 APP1_PART_TMP_SIZE = FLASH_SIZE
 APP2_PART_TMP_SIZE = FLASH_SIZE
-
+size=len(SIZE_LIST)
 for name in NAME_LIST :
+    if PART_IDX>size : break
     PART_IDX += 1
     if name == "kernel" or name == "os" :
         if KERNEL_PART_TMP_SIZE > (int(SIZE_LIST[PART_IDX]) * 1024) :
```

Comment:
Request you to mention the issue https://github.com/Samsung/TizenRT/issues/5845 for which this fix is intended in the PR description